the-experts-fe/src/App.js

```diff
@@ -2,21 +2,21 @@ import React, { useState, useEffect } from "react";
 import "./App.css";
 import NavBar from "./components/NavBar/NavBar";
 import RegisterationForm from "./components/registerationForm/registerationForm";
-import ContractorProfile from "./components/ContractorProfile/ContractorProfile"
+import ContractorProfile from "./components/ContractorProfile/ContractorProfile";
 import { Route, Switch } from "react-router-dom";
 import Home from "./components/home/home";
 
-
 export default function App() {
   return (
     <React.Fragment>
       <NavBar firstParm="Login" secondParam="Register" />
-        <Switch className="App">
-          <Route path="/register" component={RegisterationForm}></Route>
-          <Route path="/ContractorProfile/:id"  component={ContractorProfile} />}/>
-          <Route exact path="/" component={Home}></Route>
-        </Switch>
-        <ReviewForm></ReviewForm>
+      <Switch className="App">
+        <Route path="/register" component={RegisterationForm}></Route>
+        <Route path="/ContractorProfile/:id" component={ContractorProfile} />
+        }/>
```

Comment:
`}/>` looks like it was accidentally left in, as your `<Route />` is self closing. Can you remove it?